packages/inventory/src/components/table/InventoryList.js

```diff
@@ -41,44 +40,18 @@ const ContextInventoryList = ({ showHealth, onRefreshData, ...props }) => {
 
     return (<Grid gutter="sm" className="ins-inventory-list">
         <GridItem span={ 12 }>
-            <InventoryEntityTable { ...props } />
+            <InventoryEntityTable { ...props } onRefreshData={onRefreshData} />
         </GridItem>
     </Grid>);
 };
 
 /**
  * Component that consumes active filters and passes them down to component.
  */
-const InventoryList = React.forwardRef(({ hasAccess, getEntities, hideFilters, ...props }, ref) => {
-    const dispatch = useDispatch();
+const InventoryList = React.forwardRef(({ hasAccess, onRefreshData, ...props }, ref) => {
     const activeFilters = useSelector(({ entities: { activeFilters } }) => activeFilters);
 
-    /**
-     * If conumer wants to change data they can call this function via component ref.
-     * @param {*} options new options to be applied, like pagination, filters, etc.
-     */
-    const onRefreshData = (options = {}) => {
-        const { page, perPage, items, hasItems, sortBy, activeFilters, showTags, customFilters } = props;
-        dispatch(
-            loadSystems(
-                {
-                    page,
-                    perPage,
-                    items,
-                    hasItems,
-                    sortBy,
-                    activeFilters,
-                    hideFilters,
-                    ...customFilters,
-                    ...options
-                },
-                showTags,
-                getEntities
-            )
-        );
-    };
-
-    if (ref) { ref.current = { onRefreshData }; }
+    if (ref) { ref.current = { onRefreshData: (params, disableRefresh = true) => onRefreshData(params, disableRefresh) }; }
```

Comment:
I would span this over a couple lines
```suggestion
    if (ref) {
      ref.current = {
        onRefreshData: (params, disableRefresh = true) => onRefreshData(params, disableRefresh)
      };
    }
```

Reply:
I want to move this logic to the upper component in the follow-up pr.

(And also, we should set eslint if really want to change formatting. 😄  )